tools/create_cve_json.py

```diff
@@ -0,0 +1,125 @@
+import argparse
+import json
+import aiohttp
+import asyncio
+import sys
+from cvsslib import cvss31, calculate_vector
+
+HEADER = """
+z/OS Open Tools Vulnerability JSON Generator
+
+This tool fetches z/OS Open Tools releases and generates a list of tools and their associated CVEs using the osv.dev API.
+"""
+
+def calculate_severity(cvss_score):
+    if cvss_score >= 9.0:
+        return "CRITICAL"
+    elif cvss_score >= 7.0:
+        return "HIGH"
+    elif cvss_score >= 4.0:
+        return "MEDIUM"
+    else:
+        return "LOW"
+
+parser = argparse.ArgumentParser(description=HEADER)
+parser.add_argument('--verbose', action='store_true', help='Enable verbose output')
+parser.add_argument('--output-file', dest='output_file', required=True, help='The full path to store the JSON file to')
+args = parser.parse_args()
+
+# Use the zopen_releases.json cache to iterate on releases
+url = "https://raw.githubusercontent.com/ZOSOpenTools/meta/main/docs/api/zopen_releases.json"
+if args.verbose:
+    print("Fetching data from:", url)
+
+# Use the OSV API: 
+async def fetch_cves(session, commit):
+    url = "https://api.osv.dev/v1/query"
+    data = {
+        "commit": commit
+    }
+    async with session.post(url, json=data, ssl=False) as response:
```

Comment:
Future: add a todo comment so that we look into enabling SSL. I see it's set to false here